src/main/java/uk/gov/hmcts/reform/idam/api/fr/am/authenticate/AuthenticationChainApi.java

```diff
@@ -79,13 +71,9 @@
 
             "X-Origin-IP: {xOriginIP}",
 
-            "X-Redirect-URI: {xRedirectURI}",
-
-            "X-Admin-Username: {xAdminUsername}",
-
-            "X-Admin-Password: {xAdminPassword}"
+            "X-Redirect-URI: {xRedirectURI}"
     })
-    AuthenticationChain authenticate(@Param("myRealm") String myRealm, @Param("xOpenAMCustUsername") String xOpenAMCustUsername, @Param("xOpenAMCustPassword") String xOpenAMCustPassword, @Param("xOriginIP") String xOriginIP, @Param("xRedirectURI") String xRedirectURI, @Param("xAdminUsername") String xAdminUsername, @Param("xAdminPassword") String xAdminPassword, @QueryMap(encoded=true) Map<String, Object> queryParams);
+    AuthenticationChain authenticate(@Param("myRealm") String myRealm, @Param("xOriginIP") String xOriginIP, @Param("xRedirectURI") String xRedirectURI, @Param("xOpenAMCustUsername") String xOpenAMCustUsername, @Param("xOpenAMCustPassword") String xOpenAMCustPassword, @QueryMap(encoded=true) Map<String, Object> queryParams);
```

Comment:
interesting I dont remember seeing this in te docs before `xOpenAMCustUsername`

Reply:
Yeah, we need to use a custom header name because FR has a bug which they hadn't fixed in a few years....